MIGRATION.md

```diff
@@ -82,11 +83,43 @@ Storybook now uses webpack 4. If you have a [custom webpack config](https://stor
 
 Storybook now uses Babel 7. There's a couple of cases when it can break with your app:
 
-* If you aren't using Babel yourself, and don't have .babelrc, install following dependencies:
+- If you aren't using Babel yourself, and don't have .babelrc, install following dependencies:
   ```
   npm i -D @babel/core babel-loader@next
   ```
-* If you're using Babel 6, make sure that you have direct dependencies on `babel-core@6` and `babel-loader@7`.
+- If you're using Babel 6, make sure that you have direct dependencies on `babel-core@6` and `babel-loader@7` and that you have a `.babelrc` in your project directory.
+
+### Create-react-app
+
+If you are using `create-react-app` (aka CRA), you may need to do some manual steps to upgrade, depending on the setup.
+
+- `create-react-app@1` may require manual migrations.
+  - If you're adding storybook for the first time, it should just work: `storybook init` should add the correct dependencies.
+  - If you've upgrading an existing project, your `package.json` probably already uses Babel 6, making it incompatible with `@storybook/react@4` which uses Babel 7. There are two ways to make it compatible, each of which is spelled out in detail in the next section:
+    - Upgrade to Babel 7 if you are not dependent on Babel 6-specific features.
+    - Migrate Babel 6 if you're heavily dependent on some Babel 6-specific features).
+- `create-react-app@2` should be compatible as is, since it uses babel 7.
+
+#### Upgrade CRA1 to babel 7
+
+```
+yarn remove babel-core babel-runtime
+yarn add @babel/core babel-loader --dev
+```
+
+#### Migrate CRA1 while keeping babel 6
+
+```
+yarn add babel-loader@7
+```
+
+Also make sure you have a `.babelrc` in your project directory. You probably already do if you are using Babel 6 features (otherwise you should consider upgrading to Babel 7 instead). If you don't have one, here's a simple one that works:
+
+```json
+{
+  "presets": ["env", "react"]
```

Comment:
Please mention that user should `yarn add babel-preset-env babel-preset-react` in that case